### ISSUE TITLE
Samples with warning icon should have ALT text to explain you need to sign in

### ISSUE THREAD

**Issue by jthake-msft** [microsoftgraph/microsoft-graph-explorer-v2#264]
When you hover over a sample in Graph Explorer Preview, the warning sign icon changes for the mouse cursor. But there is no ALT text to explain WHY you get the warning. Can we please add ALT text to state that "Please sign into an account to try this sample".
